pulsar-broker/src/test/java/org/apache/pulsar/broker/service/ReplicatorRateLimiterTest.java

```diff
@@ -145,6 +145,11 @@ public void testReplicatorRatePriority() throws Exception {
                 assertEquals(topic.getReplicators().values().get(0).getRateLimiter().get().getDispatchRateOnMsg(), 100));
         assertEquals(topic.getReplicators().values().get(0).getRateLimiter().get().getDispatchRateOnByte(),
                 200L);
+        //restore config
+        config1.setSystemTopicEnabled(false);
```

Comment:
We should not do this way.
In case we change the default value here we are going to set arbitrary values.
Can you find a way to reset those values to the values that were set at the beginning of the test?

Reply:
OK, I abstracted some methods